index.html

```diff
@@ -165,8 +165,34 @@ <h4>Skills</h4>
             <li>Gitlab</li>
             <li>Terminal</li>
         </ul>
-
+    </section>
+    <section class="content">
+        <h3>
+            I'm always interested in hearing about new projects, so if you'd 
+            like to chat please get in touch.
+        </h3>
     </section>
 
+    <form action="https://formspree.io/f/xdobraqv" method="post" class="form1">
+        <input name="input1" type="text" required class="name" placeholder=" Full name" maxlength="30">
+        <div class="div-intermediate1"></div>
+        <input name="input2" type="email" required class="email" placeholder="Email address">
```

Comment:
- [ ] `required`
- [ ] Please kindly add the Message input  maxlength="500" attribute.